pkg/client/unversioned/request.go

```diff
@@ -648,12 +687,70 @@ func (r *Request) Stream() (io.ReadCloser, error) {
 	}
 }
 
+type URLBackoff struct {
```

Comment:
its definetely easy to   move it.... 
...but more feedback needed (i.e. where ?) .  because there isn't really a natural place for it to live IMO